addons/l10n_es_edi_tbai/models/account_move.py

```diff
@@ -73,6 +73,12 @@ class AccountMove(models.Model):
         "Valor Añadido. Artículo 80. Modificación de la base imponible.",
         copy=False,
     )
+    l10n_es_tbai_reversed_ids = fields.Many2many(
```

Comment:
You can't do that in a stable branch :/

If you manually run the l10n builds, the upgrade should fail

Reply:
However, the field is needed. Customers cannot wait to upgrade to be able to refund invoices...

What could we do? Should I push a new auto-installable module?

Reply:
Yes if it's absolutely necessary and can't be computed, you should do a temporary auto-installed module that will be deleted in master.

Reply:
Well, not master. This is a backport from something that landed in 17.3, so it can be deleted in that version already.

Attended. Could you review again please?